zerver/tests/test_events.py

```diff
@@ -258,6 +260,8 @@ def match_states(self, state1, state2):
         def normalize(state):
             # type: (Dict[str, Any]) -> None
             state['realm_users'] = {u['email']: u for u in state['realm_users']}
+            for u in state['subscriptions']:
+                u['subscribers'].sort()
```

Comment:
I went ahead and merged this hunk as its own commit.  @krtkmj in the future, please separate things like this that are changed to existing out as their own commit, not as part of a larger commit.